Mathlib/Logic/Small/Set.lean

```diff
@@ -41,6 +41,9 @@ instance small_image2 (f : α → β → γ) (s : Set α) (t : Set β) [Small.{u
   rw [← Set.image_uncurry_prod]
   infer_instance
 
+theorem small_univ_iff : Small.{u} (@Set.univ α) ↔ Small.{u} α :=
```

Comment:
Is the reverse implication already a lemma?

Reply:
No, but it seems worth adding as an instance.